dawn/test/unit-test/dawn/Optimizer/Passes/TestIntervalPartitioner.cpp

```diff
@@ -0,0 +1,94 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
What was the decision process to start from SIR here?

It feels like we should be able to start form a generated / deserialized IIR and just call the one pass on it?